Index: local.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## This file must *NOT* be checked into Version Control Systems,\r\n# as it contains information specific to your local configuration.\r\n#\r\n# Location of the SDK. This is only used by Gradle.\r\n# For customization when using a Version Control System, please read the\r\n# header note.\r\n#Mon Apr 24 20:39:15 PET 2023\r\nsdk.dir=C\\:\\\\Users\\\\Valeria\\\\AppData\\\\Local\\\\Android\\\\Sdk\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/local.properties b/local.properties
--- a/local.properties	(revision 285598eb6513f0843e2eade714333fe1a839fcf2)
+++ b/local.properties	(date 1682390646797)
@@ -4,5 +4,5 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Mon Apr 24 20:39:15 PET 2023
-sdk.dir=C\:\\Users\\Valeria\\AppData\\Local\\Android\\Sdk
+#Mon Apr 24 21:44:06 COT 2023
+sdk.dir=C\:\\Users\\ASUS\\AppData\\Local\\Android\\Sdk
